### ISSUE TITLE
code issue

### ISSUE THREAD

**Issue by theDyingofLight** [yanghoonkim/NQG_ASs2s#10]
Hi, thank you for your work. I find out that the 176th line in model.py: ` p_s = tf.nn.softmax(tf.matmul(m, last_attention), name = 'p_s')` the softmax is on the last dim but it should be that dim=1. I am not sure about the issue.
